test/test_jsonPath.js

```diff
@@ -8,7 +8,7 @@ var data = {
         title: "Sayings of the Century",
         price: 8.95,
         publishDate:"2015-01-01",
-        disabled:false
+        disabled: false
```

Comment:
Weird that you (or your IDE) fixed this one, but not the missing space on line 10.